src/inspector/view/qml/MuseScore/Inspector/notation/chordsymbols/ChordSymbolSettings.qml

```diff
@@ -36,13 +36,22 @@ Column {
 
     spacing: 16
 
-    InspectorPropertyView {
-        titleText: qsTrc("inspector", "Chord symbol style")
-
-        ChordSymbolStyleSettings {
-            id: styleSettings
-            width: parent.width
-            chordStylesModel: root.model ? root.model.chordStylesModel : null
-        }
+    StyledTextLabel{
+        text: qsTrc("inspector", "Chord symbol style")
+    }
+
+    ChordSymbolStyleSettings {
+        id: styleSettings
+        width: parent.width
+        chordStylesModel: root.model ? root.model.chordStylesModel : null
+    }
+
+    FlatButton {
+        width: parent.width
+        height: 30
+
+        text: qsTrc("inspector", "Edit styles NW")
```

Comment:
What does NW stand for?

Reply:
"Not Working" :) 
I have not implemented the functionality for that button yet. That's why.

Reply:
;-)